frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -72,6 +72,11 @@ const Map: React.FC<MapProps> = ({
   const [baseLayers, setBaseLayers] = useState<BaseLayer[]>([]);
   const [activeBasemap, setActiveBasemap] = useState<BaseLayer | null>(null);
 
+  //do not jump to map coordinates if we have an existing map but no parcel details.
+  if (mapRef.current && !selectedProperty?.parcelDetail) {
+    lat = (mapRef.current.props.center as Array<number>)[0];
+    lng = (mapRef.current.props.center as Array<number>)[1];
+  }
```

Comment:
I'm okay if this PR gets merged as it has already been approved but the above doesn't seem to fix the jumping around on the map. I will post a video shortly.

Reply:
We talked about this in rocketchat, this PR fixes two issues:
1) The map moving when the popup is closed.
2) The map only opening popups once.

This does not fix the issue assigned to Goeme PIMS-1549 (and isn't intended to).